examples/asr/transcribe_speech.py

```diff
@@ -178,6 +182,11 @@ class TranscriptionConfig:
     # key for groundtruth text in manifest
     gt_text_attr_name: str = "text"
 
+    # Use model's transcribe() function instead of transcribe_partial_audio()
+    # Only use transcribe_partial_audio() when the audio is too long to fit in memory
+    # and your model's transcribe() function does not support processing manifests with audio offsets
+    use_model_transcribe: bool = False
```

Comment:
CC @stevehuang52

Reply:
I agree that `use_model_transcribe` is not a good flag and I too want to remove it. But since the canary model's transcribe function takes manifest as input, which is very different from all other models' transcribe() function, we need to have a special case for multi-task AED model. I will remove this flag, and add another `if` condition to skip the `prepare_audio_data` function for multi-task AED models